pages/_prefix/explore/items.vue

```diff
@@ -1,8 +1,5 @@
 <template>
-  <div>
-    <Items v-if="redesign" />
-    <Gallery v-else class="container" />
-  </div>
+  <Items />
```

Comment:
is it done already?
can you wrap `Items` with `<div>`? (might be needed to add container here #5089)

Reply:
Yes, I'm ok with that (https://github.com/kodadot/nft-gallery/issues/5069#issuecomment-1458381445)